test/functional/feature_segwit.py

```diff
@@ -581,7 +580,7 @@ def create_and_mine_tx_from_txids(self, txids, success=True):
         tx = CTransaction()
         for i in txids:
             txtmp = CTransaction()
-            txraw = self.nodes[0].getrawtransaction(i)
+            txraw = self.nodes[0].getrawtransaction(i, 0, txs_mined[i])
```

Comment:
nit: the second argument to `getrawtransaction()` is a bool, so this should be:

```
txraw = self.nodes[0].getrawtransaction(i, False, txs_mined[i])
```

Even better, use named args and drop the unused param to make it obvious what the params are for:

```
txraw = self.nodes[0].getrawtransaction(txid=i, blockhash=txs_mined[i])
```